exit 1

run:
  failed

test:
  starting
  exit: exit 1
